libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -1046,6 +1046,7 @@
   <!--
     Node: <place2d>
     Transform incoming UV texture coordinates for 2D texture placement.
+    Sequence of transformation (in the order of application): -pivot, scale, rotate, translate, pivot.
   -->
   <nodegraph name="NG_place2d_vector2" nodedef="ND_place2d_vector2">
```

Comment:
Instead of continuing to use the `nodedef=` association on the nodegraph, I'd suggest to use the newer "implementation<nodgraph, nodedef>" association introduced for 1.39. Standard surface has an example of this.
This is supported by USD already if there are any concerns on that front.

Reply:
FYI @vernalchen , I think th implementation example mentioned by Bernard is here: https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/bxdf/standard_surface.mtlx#L106